client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -205,7 +224,210 @@ export const useCommandsArrayWpcom = ( {
+			label: __( 'Open hosting configuration' ),
+			searchLabel: __( 'open hosting configuration' ),
+			context: 'Opening hosting configuration',
+			callback: setStateCallback( 'openHostingConfiguration' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/hosting-config/${ site.slug }` );
+				},
+				filter: ( site: SiteExcerptData ) => ! isP2Site( site ) && ! isNotAtomicJetpack( site ),
+			},
+			icon: hostingConfigIcon,
+		},
+		{
+			name: 'openPHPmyAdmin',
+			label: __( 'Open phpMyAdmin' ),
+			searchLabel: __( 'open phpMyAdmin' ),
+			context: 'Opening phpMyAdmin',
+			callback: setStateCallback( 'openPHPmyAdmin' ),
+			siteFunctions: {
+				onClick: async ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					await openPHPmyAdmin( site.ID );
+				},
+				filter: ( site: SiteExcerptData ) => site?.is_wpcom_atomic,
+			},
+			icon: pageIcon,
+		},
+		{
+			name: 'manageStagingSites',
+			label: __( 'Manage staging sites' ),
+			searchLabel: __( 'manage staging sites' ),
+			context: 'Managing staging sites',
+			callback: setStateCallback( 'manageStagingSites' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/hosting-config/${ site.slug }#staging-site` );
+				},
+				filter: ( site: SiteExcerptData ) => site?.is_wpcom_atomic,
+			},
+			icon: toolIcon,
+		},
+		{
+			name: 'managePHPVersion',
+			label: __( 'Manage PHP version' ),
+			searchLabel: __( 'manage PHP issue' ),
+			context: 'Managing PHP version',
+			callback: setStateCallback( 'managePHPVersion' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/hosting-config/${ site.slug }#web-server-settings` );
+				},
+				filter: ( site: SiteExcerptData ) => site?.is_wpcom_atomic,
+			},
+			icon: toolIcon,
+		},
+		{
+			name: 'manageCacheSettings',
+			label: __( 'Manage cache settings' ),
+			searchLabel: __( 'manage cache settings' ),
+			context: 'Managing cache settings',
+			callback: setStateCallback( 'manageCacheSettings' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/hosting-config/${ site.slug }#cache` );
+				},
+				filter: ( site: SiteExcerptData ) => site?.is_wpcom_atomic,
+			},
+			icon: toolIcon,
+		},
+		{
+			name: 'manageAdminInterfaceStyle',
+			label: __( 'Manage admin interface style' ),
+			searchLabel: __( 'manage admin interface style' ),
+			context: 'Managing admin interface style',
+			callback: setStateCallback( 'manageAdminInterfaceStyle' ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/hosting-config/${ site.slug }#sitewpadmin-card` );
```

Comment:
It seems that this has been renamed on trunk to `admin-interface-style`

Reply:
Done!